### PR TITLE
Docker inspect as command

### PR DESCRIPTION
**Pull request by Yuki-Inoue**
From the `docker inspect` result, create the shell-command which would reproduce the original `docker run` command.

### CODE REVIEW

File: docker-containers.el [hunk 1]
```diff
@@ -200,8 +200,9 @@ Remove the volumes associated with the container when VOLUMES is set."
              (parsed (json-read-from-string json))
              (commands
               (docker-containers-convert-container-info-to-command parsed)))
+        (pop-to-buffer "*docker result*" )
```
Comment (Silex) [inline docker-containers.el]: Remove ending space please ;)

**Comment by Silex** [comment]
Looks interesting... did you test it well?

**Comment by Yuki-Inoue** [comment]
@Silex I assume you're afraid the effect (the bugs) caused by refactoring the common functionalities (the docker-utils functions.)  Maybe I should list up the tests I've done.  I'll make a checklist.

**Comment by Silex** [comment]
@Yuki-Inoue: I trust you, if you say you tested I'll just merge it

I tend to push code and test it only very quickly without testing much for regressions in other areas :-)

**Comment by Silex** [comment]
Oh well, we'll see if there are bugs :stuck_out_tongue_closed_eyes:

**Comment by Yuki-Inoue** [comment]
:)

**Pull request merged**